src/package.test.ts

```diff
@@ -465,27 +467,58 @@ describe('package', () => {
             changelogPath,
           }),
           newVersion: '2.0.0',
-          shouldUpdateChangelog: true,
         };
-        when(jest.spyOn(autoChangelog, 'updateChangelog'))
-          .calledWith({
-            changelogContent: 'existing changelog',
-            currentVersion: '2.0.0',
-            isReleaseCandidate: true,
-            projectRootDirectory: sandbox.directoryPath,
-            repoUrl: 'https://repo.url',
-            tagPrefixes: ['package@', 'v'],
-          })
-          .mockResolvedValue('new changelog');
-        await fs.promises.writeFile(changelogPath, 'existing changelog');
+
+        await fs.promises.writeFile(
+          changelogPath,
+          outdent`
+          # Changelog
+          All notable changes to this project will be documented in this file.
+
+          The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+          and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+          ## [Unreleased]
+          ### Uncategorized
+          - Add \`isNewFunction\` ([#2](https://repo.url/compare/package/pull/2))
+
+          ## [1.0.0] - 2020-01-01
+          ### Changed
+          - Something else
+
+          [Unreleased]: https://repo.url/compare/package@2.0.0...HEAD
+          [1.0.0]: https://repo.url/releases/tag/package@1.0.0
+        `,
+        );
 
         await updatePackage({ project, packageReleasePlan });
 
         const newChangelogContent = await fs.promises.readFile(
           changelogPath,
           'utf8',
         );
-        expect(newChangelogContent).toBe('new changelog');
+
+        expect(newChangelogContent).toBe(outdent`
+        # Changelog
+        All notable changes to this project will be documented in this file.
+
+        The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+        and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+        ## [Unreleased]
+
+        ## [2.0.0]
+        ### Uncategorized
+        - Add \`isNewFunction\` ([#2](https://repo.url/compare/package/pull/2))
+
+        ## [1.0.0] - 2020-01-01
+        ### Changed
+        - Something else
+
+        [Unreleased]: https://repo.url/compare/package@2.0.0...HEAD
+        [2.0.0]: https://repo.url/compare/package@1.0.0...package@2.0.0
+        [1.0.0]: https://repo.url/releases/tag/package@1.0.0
+      `);
```

Comment:
Similar as above. Perhaps we want:

``` suggestion
        expect(newChangelogContent).toBe(outdent`
          # Changelog
          All notable changes to this project will be documented in this file.
          The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
          and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
          ## [Unreleased]
          ## [2.0.0]
          ### Uncategorized
          - Add \`isNewFunction\` ([#2](https://repo.url/compare/package/pull/2))
          ## [1.0.0] - 2020-01-01
          ### Changed
          - Something else
          [Unreleased]: https://repo.url/compare/package@2.0.0...HEAD
          [2.0.0]: https://repo.url/compare/package@1.0.0...package@2.0.0
          [1.0.0]: https://repo.url/releases/tag/package@1.0.0
        `);

Reply:
test fail on this case as parseChangelog does add these empty lines